libs/judicial-system/types/src/lib/tables/caseTable.ts

```diff
@@ -4,124 +4,122 @@ import {
         return CaseTableType.DISTRICT_COURT_INDICTMENTS_RECEIVED
-      case 'sakamal-i-vinnslu':
+      case CaseTableRoutes.INDICTMENT_IN_PROGRESS:
         return CaseTableType.DISTRICT_COURT_INDICTMENTS_IN_PROGRESS
-      case 'sakamal-i-fragangi':
+      case CaseTableRoutes.INDICTMENT_FINALIZING:
         return CaseTableType.DISTRICT_COURT_INDICTMENTS_FINALIZING
-      case 'afgreidd-sakamal':
+      case CaseTableRoutes.INDICTMENT_COMPLETED:
         return CaseTableType.DISTRICT_COURT_INDICTMENTS_COMPLETED
     }
   }
   if (isCourtOfAppealsUser(user)) {
     switch (route) {
-      case 'mal-i-vinnslu':
+      case CaseTableRoutes.IN_PROGRESS:
         return CaseTableType.COURT_OF_APPEALS_IN_PROGRESS
-      case 'afgreidd-mal':
+      case CaseTableRoutes.COMPLETED:
         return CaseTableType.COURT_OF_APPEALS_COMPLETED
     }
   }
 
   if (isPrisonAdminUser(user)) {
     switch (route) {
-      case 'virk-mal':
+      case CaseTableRoutes.ACTIVE:
         return CaseTableType.PRISON_ADMIN_ACTIVE
-      case 'lokid':
+      case CaseTableRoutes.DONE:
         return CaseTableType.PRISON_ADMIN_DONE
-      case 'mal-til-fullnustu':
+      case CaseTableRoutes.SENT_TO_PRISON:
         return CaseTableType.PRISON_ADMIN_INDICTMENT_SENT_TO_PRISON_ADMIN
-      case 'skradir-domar':
+      case CaseTableRoutes.REGISTERED_RULING:
         return CaseTableType.PRISON_ADMIN_INDICTMENT_REGISTERED_RULING
     }
   }
 
   if (isPrisonStaffUser(user)) {
     switch (route) {
-      case 'virk-mal':
+      case CaseTableRoutes.ACTIVE:
         return CaseTableType.PRISON_ACTIVE
-      case 'lokid':
+      case CaseTableRoutes.DONE:
         return CaseTableType.PRISON_DONE
     }
   }
 
   if (isPublicProsecutionOfficeUser(user)) {
     switch (route) {
-      case 'ny-mal':
+      case CaseTableRoutes.NEW:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_NEW
-      case 'mal-i-yfirlestri':
+      case CaseTableRoutes.IN_REVIEW:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_IN_REVIEW
-      case 'yfirlesin-mal':
+      case CaseTableRoutes.REVIEWED:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_REVIEWED
-      case 'frestur-lidinn':
+      case CaseTableRoutes.APPEALED_EXPIRED:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_APPEAL_PERIOD_EXPIRED
-      case 'mal-i-fullnustu':
+      case CaseTableRoutes.SENT_TO_PRISON:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_SENT_TO_PRISON_ADMIN
-      case 'afryjud-mal':
+      case CaseTableRoutes.INDICTMENT_APPEALED:
         return CaseTableType.PROSECUTORS_OFFICE_INDICTMENT_APPEALED
     }
   }
-}
-
-interface CaseTable {
-  title: string
-  hasMyCasesFilter: boolean
-  columnKeys: CaseTableColumnKey[]
-  columns: CaseTableColumn[]
-}
 
-const pickColumns = (
-  keys: CaseTableColumnKey[],
-): CaseTableColumnMap[CaseTableColumnKey][] => {
-  return keys.map((key) => caseTableColumns[key])
+  if (isProsecutionUser(user)) {
```

Comment:
Differenct prosecution roles have access to different tables. Please consider splitting this into two separate conditions, one for is prosecutors and one for prosecutor representatives. Please consider creating helper methods to check for these similar to `isProsecutionUser`.